### PR TITLE
Alien Brain Hemorrhage

### PR DESCRIPTION
**Pull request by xhalin01**
adding a new recipe for alien brain hemorrhage

### CODE REVIEW

File: src/recipes/alien-brain-hemorrhage.json [hunk 1]
```diff
@@ -0,0 +1,47 @@
+{
+    "name": "Alien brain hemorrhage",
```
Comment (alfg) [inline src/recipes/alien-brain-hemorrhage.json:2]:
```suggestion
    "name": "Alien Brain Hemorrhage",

```

File: src/recipes/alien-brain-hemorrhage.json [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+        }
+    ],
+    "directions": [
+        "Fill a shot glass with Peach Schnapps.",
+        "Put Bailey’s Irish Cream carefuly on top using spoon.",
+        "Slowly add a dash of of Blue Curacao.",
+        "Finish with a dash of grenadine."
+    ],
+    "image": "alien-brain-hemorrhage.jpg",
+    "source": "<source url if available, otherwise remove this line>",
```
Comment (alfg) [inline src/recipes/alien-brain-hemorrhage.json:34]:
```suggestion
```

Can remove this line if no source provided.

**Review by alfg — CHANGES_REQUESTED** [review]
Nice one! Lol. I left 2 suggestions before we can merge. Thanks.

**Comment by xhalin01** [comment]
Thanks for your suggestions, I updated the source and name of the drink. It should be ready to **merge**.

**Comment by alfg** [comment]
Thank you!

**Comment by xhalin01** [comment]
> Thank you!

You are welcome.

**Pull request merged**